content/en/Platform Deep Dive/Scans/scans.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Scans"
+linkTitle: "Scans"
+weight: 50
+description: >
+  A scan represents a test run against a specific target
+---
+
+## Statuses
+
+A scan can have the following states:
+- Canceled
+- Canceling
+- Completed
+- Completed With Errors
+- Failed
+- Finishing Up
+- Paused
+- Pausing
+- Queued
+- Resuming
+- Running
+- Under Review
+
+To start a scan, you can either **start** it immediately or **schedule** it for a later point in time. This can even be on a recurring basis.
```

Comment:
We can link the scheduling part to https://docs.cobalt.io/platform-deep-dive/scans/#scan-scheduling